recipes-wigwag/mbed-fcce/mbed-fcce_4.7.1.bb

```diff
@@ -55,16 +55,16 @@ do_compile() {
 
     python3 pal-platform/pal-platform.py -v deploy --target=Yocto_Generic_YoctoLinux_mbedtls generate
 
-    # applying 0001-fix-arm_uc_pal_linux_extensions.manual_patch
-    git -C mbed-cloud-client apply ../../0001-fix-arm_uc_pal_linux_extensions.manual_patch
+    if [ "${MBED_EDGE_CORE_CONFIG_PARSEC_TPM_SE_SUPPORT}" = "ON" ]; then
 
-    if [ ${MBED_EDGE_CORE_CONFIG_PARSEC_TPM_SE_SUPPORT} == "ON" ]; then
+        git -C mbed-cloud-client apply ${WORKDIR}/0001-fix-arm_uc_pal_linux_extensions.manual_patch
+        git apply ${WORKDIR}/0001-fix-mcc_common_setup.manual_patch
 
         # Manually adding the parsec-se-driver Middleware
         cp -R ${WORKDIR}/parsec-se-driver ${S}/pal-platform/Middleware/parsec_se_driver/parsec_se_driver
 
         # Place the precompiled parsec-se-driver static library
-        if [ ${MBED_EDGE_CMAKE_BUILD_TYPE} == "Debug" ]; then
+        if [ "${MBED_EDGE_CMAKE_BUILD_TYPE}" = "Debug" ]; then
```

Comment:
@JanneKiiskila  This is done because in bitbake it is more likely that is using `dash` instead of `bash` - https://www.yoctoproject.org/docs/2.3.4/bitbake-user-manual/bitbake-user-manual.html#shell-functions

This rule will also work in `bash` according to - https://stackoverflow.com/a/1089852/12488057

Reply:
Yep, if it's executed in shell then a single equals sign is correct.

Reply:
Which then probably means that some other places are now wrong due to double-equals being used.

https://github.com/PelionIoT/meta-pelion-edge/blob/master/classes/goarch.bbclass#L107

Reply:
Nope, it's confusing. That's a def function, so it's actually Python and then it should use `==`.